packages/builder/src/ipfs.ts

```diff
@@ -145,7 +145,7 @@ export async function writeIpfs(
 
   let result: AxiosResponse<any, any>;
   try {
-    result = await axios.post(ipfsUrl.replace('+ipfs', '') + '/api/v0/add?local=true', formData, {
```

Comment:
Unfortunately we cannot remove this line; it causes breakage on `ipfs-cluster` servers in misc circumstances. Does the request still work with infura with both query string options?

Reply:
[8c31779](https://github.com/usecannon/cannon/pull/844/commits/8c317791550e0ca0e5d068edf21d48fde8f84458)